### ISSUE TITLE
Shoulder cross-body adduction test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11009]
**Preferred term label:**
Shoulder cross-body adduction test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain in the ipsilateral acromioclavicular joint when patient straightens their arm forward and reaches for the contralateral shoulder with extra manual cross body (adduction) pressure on the arm from the examiner; aka cross-arm test; aka scarf test;

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acromioclavicular (AC) joint osteoarthritis (OA)
Distal clavicle osteolysis (DCO)

**Your nano-attribution (ORCID)**
https://www.physio-pedia.com/Scarf_Test

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001340
- label: Positive shoulder cross-body adduction test
- parents:  Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: PMID:15090381
- def: A test used with suspected chronic acromioclavicular joint lesions (which are a common source of pain and disability in the shoulder). The test is considered positive with pain in the ipsilateral acromioclavicular joint when patient straightens the affected arm forward and reaches for the contralateral shoulder with extra manual cross body (adduction) pressure on the arm from the examiner.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11009
- synonyms: Positive scarf test(NONE)
